corehq/apps/users/migrations/0046_add_default_mobile_worker_role.py

```diff
@@ -0,0 +1,35 @@
+# Adds the default mobile worker role to existing projects that don't yet have one.
+from django.db import migrations
+
+from corehq.apps.domain.models import Domain
+from corehq.apps.users.models import Permissions
+from corehq.apps.users.models_role import UserRole
+from corehq.apps.users.role_utils import UserRolePresets
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _add_default_mobile_worker_role(apps, schema_editor):
+    default_mobile_worker_roles = UserRole.objects.filter(is_commcare_user_default=True)
+    for domain in Domain.get_all():
+        dmw_roles_in_domain = default_mobile_worker_roles.filter(domain=domain)
+        if len(dmw_roles_in_domain) == 0:
```

Comment:
I think my suggestion above about just getting the domains is best, but for future reference, this will perform a smaller query (it returns only a bool instead of a list of the matching role)
```suggestion
        has_dmw_role = default_mobile_worker_roles.filter(domain=domain).exists()
        if has_dmw_role:
```